addons/account/models/product.py

```diff
@@ -60,7 +60,8 @@ class ProductProduct(models.Model):
     @api.model
     def _get_tax_included_unit_price(self, company, currency, document_date, document_type,
             is_refund_document=False, product_uom=None, product_currency=None,
-            product_price_unit=None, product_taxes=None, fiscal_position=None
+            product_price_unit=None, product_taxes=None, fiscal_position=None,
+            line_taxes=None
```

Comment:
`taxes` instead of `line_taxes` because `line` is only about the implementation but there is no guarantee this will be always the case